src/containers/Landing/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+import Banner from '../../components/Banner'
+import logo_lockup from '../../assets/logo_lockup.svg'
+import holo from '../../assets/holo.svg'
+import holo_end from '../../assets/holo2.svg'
+import Footer from './Footer'
+import { H1, P } from '../../components/Typography'
+import { CenterHorizontally } from '../../components/Common'
+
+const LandingContainer = styled.div`
+  overflow-x: hidden;
+  padding: 0 auto;
+`
+
+const HoloBackground = styled.img`
+  position: absolute;
+  float: left;
+  overflow-x: hidden;
+  display: block;
+  opacity: 0.8;
+  width: 100%;
+  top: 18em;
+  ${CenterHorizontally}
+  z-index: -1;
+`
+
+const StyledLogoLockup = styled.img`
+  position: absolute;
+  top: 7em;
+  margin: 0 50.5%;
+  transform: translateX(-50%);
+  z-index: 200;
+`
+
+const StyledBanner = styled(Banner)`
+  && {
```

Comment:
What is this for?

Reply:
I was having issues styling the `Banner` component so `&& {}` was a workaround I found https://github.com/styled-components/styled-components/issues/1816